pkg/util/http.go

```diff
@@ -40,8 +40,10 @@ func CreateHTTPClient(timeout time.Duration, unsafeSsl bool) *http.Client {
 	httpClient := &http.Client{
 		Timeout: timeout,
 		Transport: &http.Transport{
-			TLSClientConfig: &tls.Config{InsecureSkipVerify: unsafeSsl},
-			Proxy:           http.ProxyFromEnvironment,
+			TLSClientConfig:   &tls.Config{InsecureSkipVerify: unsafeSsl},
+			Proxy:             http.ProxyFromEnvironment,
+			DisableKeepAlives: true,
+			IdleConnTimeout:   100 * time.Second,
```

Comment:
I think we should have the option to configure this. I mean, now always we will want to disable the keep alive, working with some scaler we could be interested on it.
IMO we should add an option to configure this, keeping the old behavior as default

Reply:
Now we can configure it by DISABLE-KEEP-ALIVES, keeping the old behavior as default

Reply:
To reduce the places where we need to change/pass the parameter, what do you think about doing a field in the package populated from the environment variable? 
There are some helpers in the same package, so you could just add another helper for getting bool envs adding this function to `env_resolver.go`:
```go
func ResolveOsEnvBool(envName string, defaultValue bool) (bool, error) {
	valueStr, found := os.LookupEnv(envName)

	if found && valueStr != "" {
		return strconv.ParseBool(valueStr)
	}

	return defaultValue, nil
}
```
and then, use it in `http.go` like:
```go
var disableKeepAlives bool

func init() {
	var err error
	disableKeepAlives, err = ResolveOsEnvBool("KEDA_HTTP_DISABLE_KEEP_ALIVE", false)
	if err != nil {
		disableKeepAlives = false
	}
}
``` 

With this approach, you don't need to pass the value from the main.go until here through all the places.
WRT environment name, please use something like `KEDA_HTTP_XXXXX`. I suggest to use `KEDA_HTTP_DISABLE_KEEP_ALIVE` but if you prefer other name following the pattern, it's okey from my side

Reply:
@JorTurFer, I have change it, please review again, thanks.